app/(dashboard)/profile/page.tsx

```diff
@@ -24,6 +25,9 @@ const ProfilePage = async () => {
               <h2 className='italic'>{userProfile?.data.username}</h2>
               <LogOutButton>LOG OUT</LogOutButton>
             </div>
+            <p className='text-md mt-5'>
```

Comment:
Move this to the edit menu?

Reply:
Moved it and made it into a `ButtonRounded` like the other because why not do that!

![Screenshot 2024-07-16 at 18 58 01](https://github.com/user-attachments/assets/96eb15ed-6fd7-4864-9ede-9f728d1e5e8b)